install_config/provisioners.adoc

```diff
@@ -15,35 +15,34 @@ toc::[]
 
 An external provisioner is an application that enables dynamic provisioning for
 a particular storage provider. External provisioners can run alongside the
-provisioner plug-ins provided by {product-title} and are configured in a
-similar way using xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc[_StorageClasses_]. However, being external, they can be deployed
-and updated independently of {product-title} on an as-needed basis.
+provisioner plug-ins provided by {product-title} and are configured in a similar
+way as the *_StorageClass_* objects are configured, as described in the  xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#[Dynamic Provisioning and Creating Storage Classes] section.
+Since these provisioners are external, you can deploy and update them
+independently of {product-title}.
 
 [[provisioners-before-you-begin]]
 == Before You Begin
-
-An Ansible playbook is available to deploy and upgrade external provisioners. You
-should familiarize yourself with the
+//check later
+An Ansible playbook is available to deploy and upgrade external provisioners. Familiarize yourself with the
```

Comment:
here it is https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-cluster/openshift_provisioners.yml